src/server/tests/fetch.js

```diff
@@ -1,10 +1,9 @@
-const API = require('../util/API')
 const jsonfile = require('jsonfile')
 
 const contributors = jsonfile.readFileSync('../../assets/data/data.json')
 const contributorsList = []
 
-for (let contributor in contributors) {
+for (var contributor in contributors) {
```

Comment:
Personally, I would prefer let over var as it binds current iteration values to itself and scoping. Is there any specific reason to replace let with var?

Reply:
Yes. eslint complains of a `no-unused-vars` warning on using let. This is because let is block scoped whereas var is globally scoped so its value persists even outside the for loop. 

Another option to remove this warning could be to assign `contributor` along with `contributorsList` above and then use it in the for but I felt the current change was minimal and went ahead with it.

Reply:
@RonLek Ideally it should not happen and I just found it's resolution on [stackoverflow](https://stackoverflow.com/questions/57833013/why-am-i-getting-a-no-unused-vars-warning-in-a-for-of-loop-and-how-do-i-fix). Please upgrade eslint to v6.8.0 and babel-eslint to v10.1.0.